### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: doc/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Bibliogram works without browser JavaScript, has no ads or tracking, and doesn't urge you to sign up.
```
Comment (alexAubin) [inline doc/DESCRIPTION.md:1]:
```suggestion
An alternative Instagram front-end

Bibliogram works without browser JavaScript, has no ads or tracking, and doesn't urge you to sign up.
```

File: scripts/upgrade [hunk 2]
```diff
@@ -119,30 +119,30 @@ ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
...
+
+# #=================================================
+# # UPDATE A CONFIG FILE
+# #=================================================
+# ynh_script_progression --message="Updating a configuration file..."
+
+# ynh_add_config --template="../conf/config.js" --destination="$final_path/config.js"
+
+# chmod 600 "$final_path/config.js"
+# chown $app:$app "$final_path/config.js"
```
Comment (alexAubin) [inline scripts/upgrade:145]: Eeeeh then let's remove this block if it ain't used anymore ?

**Review by alexAubin — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**